### PR TITLE
Fix issue with read_only fields

### PR DESCRIPTION
**Pull request by mcfilib**
I recently tried to use the `read_only` fields feature, only to find it didn't work. As far as I can tell, this is the format as per the Swagger spec. Please see my comment in the original PR at https://github.com/ruby-grape/grape-swagger-entity/pull/26.

### CODE REVIEW

File: spec/grape-swagger/entities/response_model_spec.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def app
     expect(subject['definitions'].keys).to include 'Something'
     expect(subject['definitions']['Something']).to eq(
       'type' => 'object',
-      'description' => 'This returns something or an error',
+      'description' => 'This returns something',
```
Comment (mcfilib) [inline spec/grape-swagger/entities/response_model_spec.rb]: This spec was failing on a fresh pull but I fixed it anyway.

**Comment by Bugagazavr** [comment]
Hello, thank you for your contribution, and sorry for delay.

**Comment by Bugagazavr** [comment]
Thank you

**Comment by mcfilib** [comment]
No problem at all. My fault for not spotting the Travis build failing. Cheers for merging!

**Pull request merged**